tools/record.sh

```diff
@@ -113,11 +127,27 @@ done
       then
         (
           username=${id_username[$id]}
+          
           errf '[%s] stream started\a\n' "$username"
           while :
           do
             mkdir -p "$username"
-            filename=$(printf "%s/%s.ts" $username "$(date -u '+%Y_%m_%d_%H_%M_%S_(%Z)')")
+
+            filename="$username/"
+            
+            if [ "$FILENAME_TITLE" == "1" ];
+              then
+                title=$(get_stream_title $username)
+                filename+="${title}_"
```

Comment:
We need to sanitize the filename somehow, as it is, if a streamer has their title set to `../title`, the file will be created up a directory. We also have to consider special characters that are not allowed in filenames, both on Windows and Linux.